modules/gateway/gateway.go

```diff
@@ -83,12 +84,20 @@ func New(addr string, persistDir string) (g *Gateway, err error) {
 		log:        logger,
 	}
 
-	// Load the old peer list. If it doesn't exist, no problem, but if it does,
+	// Load the old node list. If it doesn't exist, no problem, but if it does,
 	// we want to know about any errors preventing us from loading it.
 	if loadErr := g.load(); loadErr != nil && !os.IsNotExist(loadErr) {
 		return nil, loadErr
 	}
 
+	// Add the bootstrap peers to the node list.
+	if build.Release != "testing" {
```

Comment:
Because then the peerManager will connect to the actual bootstrap nodes during tests. Also, some tests already assume that the node list will be empty.